### PR TITLE
WIP: Add doc skeleton

### LINKED ISSUE #1: general documentation
Following the [octave_template](https://github.com/eurobench/pi_octave_csic), we would expect to find at the root:

* README file, with general information, plus command to use the code
* License file

### PR DESCRIPTION
**Pull request by aremazeilles**
Fix #1

### CODE REVIEW

**Comment by aremazeilles** [comment]
@juritaborri I Created a template for the documentation file. Please fill the _to be done_ areas.
I think that part of your documentation with the packinfo folder could be brought here.

**Comment by aremazeilles** [comment]
> Ok, thanks for the skeleton.
> I suppose I have to fill one per each routine, is it right?

In the purpose you could detail the different algorithm present for instance, maybe extended a bit what you have in the packinfo.
The installation is common to all.
Then in the `usage section` you should provide per routine an example of how to launch each of them.

Leave blank the octave part for the moment.

**Comment by aremazeilles** [comment]
I adjusted some format in 2119f29.

I think we can start with this. Some comments ( that we could treat in other issues if you want):

* you refer several time to specific protocol. What would be in your opinion the best way to describe these protocol?
  * refer to the material in the database?
  * Have reference to a pdf like document detailing the protocol
  * as a specific doc file resuming each of the protocol?
* About the input files, we could refer to the [Eurobench data format](https://github.com/aremazeilles/eurobench_documentation/blob/master/data_format.adoc) for the generic ones (such as emg). But the platformdata is specific to your protocol(s). We should detail it somewhere. Similarly to previous point, any opinion on where to place it? 

Depending on whether we find quickly a good solution, we can solve it it in that branch, or add these points as issues, and merge that branch

**Comment by juritaborri** [comment]
> I adjusted some format in [2119f29](https://github.com/aremazeilles/beat_routine/commit/2119f29660d8dc52ab3dec20a6fd771a9714379d).
> 
> I think we can start with this. Some comments ( that we could treat in other issues if you want):
> 
> * you refer several time to specific protocol. What would be in your opinion the best way to describe these protocol?
>   
>   * refer to the material in the database?
>   * Have reference to a pdf like document detailing the protocol
>   * as a specific doc file resuming each of the protocol?
> * About the input files, we could refer to the [Eurobench data format](https://github.com/aremazeilles/eurobench_documentation/blob/master/data_format.adoc) for the generic ones (such as emg). But the platformdata is specific to your protocol(s). We should detail it somewhere. Similarly to previous point, any opinion on where to place it?
> 
> Depending on whether we find quickly a good solution, we can solve it it in that branch, or add these points as issues, and merge that branch

I am planning to write a Software Routine manual more detailed of the READme file. In such file, I would like to better explain all protocols and Pis also through the use of images. Maybe, PDF file is the most appropriate solution. 
Relatively to the PlatformData, I can arragne an explanation to add to the EurobenchData format file.

Let me know if I can proceed in this direction,

**Comment by aremazeilles** [comment]
The Platform data is specific to your protocol, so I would not bring it to the Euobench Data spec. There we should acknowledge just that we accept that protocol generate specific csv file, with a generic structure. But then, each protocol should make their own definition of the platform data. Either in a pdf document, or here, in a readme file in the code repo.

Let's merge this file for now.

I noticed that the command to launch the code you indicate are not working directly from a linux terminal. I propose to adjust it while we go and adjust these program, i.e. in other branch

File: README.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# PI BEAT
+
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+
+Copyright (TO BE DEFINED) 2020
```
Comment (aremazeilles) [inline README.md:5]: We should set the copyright

Comment (aremazeilles) [reply README.md:5]: I am happy

**Pull request merged**